.github/workflows/build.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m venv venv
-          venv/bin/pip install -r requirements.txt
+          venv/bin/pip install --no-deps --only-binary ':all:' -r requirements.txt
```

Comment:
Out of curiosity, what does this have to do with supporting Python 3.13?

Reply:
ubernostrum suggested this in his recent blog post:

https://www.b-list.org/weblog/2023/dec/07/pip-install-safely/

I tried it out with and had issues with some packages not having wheels and thus being uninstallable. I don't know if I have any evidence of that anywhere and even if I did, it's probably worth trying again.

Reply:
this step was failing for python 3.13 because it couldn't build `grpcio`. I don't think it should be trying to build that package, so i added this to see if it would make a difference. it didn't, so i'm going to close this PR for now.

Reply:
Ahhh... yuck.